keep-ui/app/workflows/builder/builder-validators.tsx

```diff
@@ -17,10 +17,11 @@ export function globalValidatorV2(
   }
 
   if (
-    !!definition?.properties && 
-    !('manual' in definition.properties) && 
-    !('interval' in definition.properties) && 
-    !('alert' in definition.properties)
+    !!definition?.properties &&
+    !definition.properties['manual'] &&
+    !definition.properties['interval'] &&
+    !definition.properties['alert'] &&
+    !definition.properties['disabled']
```

Comment:
I think this check is not needed. this is not trigger related data.
!definition.properties['disabled']